src/processIssueComment.js

```diff
@@ -68,14 +66,29 @@ async function processAddContributor({
 }
 
 async function processIssueComment({ context, commentReply }) {
+    const commentBody = context.payload.comment.body
+    const { who, action, contributions } = parseComment(commentBody)
+    const branchName = `all-contributors/add-${who}`
+    const defaultBranch = context.payload.repository.default_branch
     const repository = new Repository({
         ...context.repo(),
         github: context.github,
+        defaultBranch,
     })
+
+    try {
```

Comment:
This block should happen after fetching config IMO

Reply:
This block should happen inside the `processAddContributor` block. (For future support of other actions)